com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeUI.Drawer.cs

```diff
@@ -60,6 +61,90 @@ static void Drawer_FeatureEnableInfo(SerializedProbeVolume serialized, Editor ow
 
         static void Drawer_BakeToolBar(SerializedProbeVolume serialized, Editor owner)
         {
+            Bounds bounds = new Bounds();
+            bool foundABound = false;
+            bool performFitting = false;
+            bool performFittingOnlyOnSelection = false;
+
+            bool ContributesToGI(Renderer renderer)
+            {
+                var flags = GameObjectUtility.GetStaticEditorFlags(renderer.gameObject) & StaticEditorFlags.ContributeGI;
+                return (flags & StaticEditorFlags.ContributeGI) != 0;
+            }
+
+            void ExpandBounds(Bounds currBound)
+            {
+                if (!foundABound)
+                {
+                    bounds = currBound;
+                    foundABound = true;
+                }
+                else
+                {
+                    bounds.Encapsulate(currBound);
+                }
+            }
+
+            GUILayout.BeginHorizontal();
+            if (GUILayout.Button(EditorGUIUtility.TrTextContent("Fit to Scene"), EditorStyles.miniButton))
+            {
+                performFitting = true;
+            }
+            if (GUILayout.Button(EditorGUIUtility.TrTextContent("Fit to Selection"), EditorStyles.miniButton))
+            {
+                performFitting = true;
+                performFittingOnlyOnSelection = true;
+            }
+
+            if (performFitting)
+            {
+                if (performFittingOnlyOnSelection)
+                {
+                    var transforms = Selection.transforms;
+                    foreach (var transform in transforms)
+                    {
+                        var childrens = transform.gameObject.GetComponentsInChildren<Transform>();
+                        foreach (var children in childrens)
+                        {
+                            Renderer childRenderer;
+                            if (children.gameObject.TryGetComponent<Renderer>(out childRenderer))
+                            {
+                                bool childContributeGI = ContributesToGI(childRenderer) && childRenderer.gameObject.activeInHierarchy;
```

Comment:
Checking for activeInHierarchy is not enough, you also need to check if the renderer is enabled because TryGetComponent<> can return disabled components.

https://docs.unity3d.com/ScriptReference/Renderer-enabled.html